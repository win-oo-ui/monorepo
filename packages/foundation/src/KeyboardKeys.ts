interface KeyboardKeyTypes {
  ENTER: "Enter";
  SPACE: " ";
  DOWN_ARROW: "ArrowDown";
  UP_ARROW: "ArrowUp";
  ESC: "Escape";
}

const keyboardKeyss: KeyboardKeyTypes = {
  ENTER: "Enter",
  SPACE: " ",
  DOWN_ARROW: "ArrowDown",
  ESC: "Escape",
  UP_ARROW: "ArrowUp",
};

export default Object.freeze(keyboardKeyss);
