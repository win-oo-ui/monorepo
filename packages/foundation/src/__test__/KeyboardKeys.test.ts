import KeyboardKeys from "../KeyboardKeys";

test("Snapshop of KeyboardKeyss: Foundation", () => {
  expect(KeyboardKeys).toMatchSnapshot();
});
