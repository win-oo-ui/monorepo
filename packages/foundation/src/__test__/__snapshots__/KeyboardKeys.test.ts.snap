// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop of KeyboardKeyss: Foundation 1`] = `
{
  "DOWN_ARROW": "ArrowDown",
  "ENTER": "Enter",
  "ESC": "Escape",
  "SPACE": " ",
  "UP_ARROW": "ArrowUp",
}
`;
